newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -827,6 +859,12 @@ func flattenDashboardWidget(in *entities.DashboardWidget, pageGUID string) (stri
 			out["nrql_query"] = flattenDashboardWidgetNRQLQuery(&in.Configuration.Table.NRQLQueries)
 		}
 		out["filter_current_dashboard"] = filterCurrentDashboard
+	case "logger.log-table-widget":
```

Comment:
This is probably wrong - all the other cases here are prefixed `vis.`, and I'd imagine that's a pattern that's not meant to be broken. However, when exporting a dash that has a log table widget in it, the result looks like this:

```json
        {
          "title": "logs",
          "layout": {
            "column": 1,
            "row": 11,
            "width": 4,
            "height": 3
          },
          "linkedEntityGuids": null,
          "visualization": {
            "id": "logger.log-table-widget"
          },
          "rawConfiguration": {
            "nrqlQueries": [
              {
                "accountId": 00000,
                "query": "SELECT `log_severity`,`timestamp`,`service_name`,`level`,`message` FROM Log
              }
            ]
          }
        }
```

Reply:
This is correct, it seems logs broke the pattern 😄

Reply:
Thanks @kidk for your reply, but that doesn't really tell me the right way to handle the broken pattern here. Please advise

Reply:
From what I can tell your code is correct, so you should be all good.

Reply:
oh, gotcha. ok! nifty haha.